### PR TITLE
Fix for Image with space in name failed to display after import…

### PR DESCRIPTION
**Pull request by Alexander-Hjelm**

### CODE REVIEW

File: src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs [hunk 1]
```diff
@@ -585,7 +586,8 @@ private void CorrectImagePath(WorkItem wi, WiItem wiItem, WiRevision rev, ref st
             foreach (var att in wiItem.Revisions.SelectMany(r => r.Attachments.Where(a => a.Change == ReferenceChangeType.Added)))
             {
                 var fileName = att.FilePath.Split('\\')?.Last() ?? string.Empty;
-                if (textField.Contains(fileName))
+                var encodedFileName = HttpUtility.UrlEncode(fileName);
+                if (textField.Contains(fileName) || textField.IndexOf(encodedFileName, StringComparison.OrdinalIgnoreCase) >= 0 || textField.Contains("_thumb_" + att.AttOriginId))
```
Comment (madkoo) [inline src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs:590]: Seemes like a really random if case

**Review by madkoo — COMMENTED** [review]
Don't follow the if case :)

**Comment by madkoo** [comment]
Branch is out of date

File: src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs (hunk 1 above)
Comment (Alexander-Hjelm) [reply src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs:590]: See the original PR here: https://github.com/solidify/jira-azuredevops-migrator/pull/327

Comment (madkoo) [reply src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs:590]: alright :)

**Review by madkoo — APPROVED** [review]
Looks fine

**Pull request merged**